### PR TITLE
chore: tweaks to scripts

### PR DESCRIPTION
**Pull request by virkt25**
loopback.io deploys are failing since old files aren't removed from `@loopback/docs`. This PR:

- removes old folder before copying files from `@loopback/docs`
- Tweaks travis.yml and package.json scripts.

### CODE REVIEW

**Review by b-admike — APPROVED** [review]

**Review by shimks — APPROVED** [review]

File: .travis.yml
Comment (b-admike) [inline .travis.yml:9]: IIUC, this is to skip the scripts (`npm run fetch-readmes` and `npm test` I presume) until you get a green light right?

Comment (virkt25) [reply .travis.yml:9]: It will skip all script when a PR is made but will run them on deploy (after a PR is merged / every 24 hours). I just didn't like seeing the `No test specified` message.

Comment (virkt25) [reply .travis.yml:9]: At least that's what I'm aiming for. I'll revert this if it's not this behaviour.

**Pull request merged**
